config/connect/deployment.yaml

```
@@ -58,7 +59,8 @@ spec:
           resources:
             limits:
               memory: "128Mi"
-              cpu: "0.2"
+            requests:
+              cpu: "0.2m"
```

```suggestion
              cpu: "0.2"
```

---

Change it to `200m` to make it equivalent to `0.2`. Check [CPU resource units](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu) for additional explanation

```suggestion
      requests:
        cpu: 200m
```